lib/active_merchant/billing/gateways/bogus.rb

```diff
@@ -90,6 +90,10 @@ def void(reference, options = {})
         end
       end
 
+      def verify(credit_card, options = {})
+        authorize(100, credit_card, options)
+      end
```

Comment:
A `verify` transaction should either be a zero amount authorization or a `MultiResponse` with a low amount followed by a `void`

Reply:
Updated.